modules/grid/src/test/java/integration/DownloadFileFromGridWithCdpTest.java

```diff
@@ -30,6 +17,26 @@
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 import static org.assertj.core.api.Assumptions.assumeThat;
 
+import com.codeborne.selenide.Browser;
+import com.codeborne.selenide.Config;
+import com.codeborne.selenide.Configuration;
+import com.codeborne.selenide.ex.FileNotDownloadedError;
+import com.codeborne.selenide.impl.FileContent;
+import com.codeborne.selenide.webdriver.ChromeDriverFactory;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.openqa.selenium.Proxy;
+import org.openqa.selenium.WebDriver;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
```

Comment:
@PetroOv Please avoid massive changing order of imports. Let them rest in peace.